alf/config_util.py

```diff
@@ -386,8 +386,7 @@ def config1(config_name,
     """
     config_node = _get_config_node(config_name)
 
-    if (raise_if_used and config_node.is_used()
-            and config_node.get_effective_value() != value):
+    if raise_if_used and config_node.is_used():
```

Comment:
So the behavior will be different but IMHO in a good way.

Just FYI there might be "config-after-use-but-with-same-value" lurking in the codebase previously, and this change will make them fail. If they exists it is most likely unintentional and we should fix them.

Reply:
Right, that's exactly what I'm thinking as well. I think this behavior is preferable and we should fix all unintentional calls in our codebase moving forward. Should be relatively quick.